package.json

```
@@ -20,7 +20,8 @@
     "vue:regenerate": "nx regenerate devextreme-vue",
     "regenerate-all": "nx run-many -t regenerate",
     "lint-staged": "lint-staged",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "build-npm-packages": "node build/build-npm-packages.js"
```

Consider this renaming:
```suggestion
    "pack-all": "node build/build-npm-packages.js"
```
or better
```suggestion
    "all:pack": "node build/build-npm-packages.js"
```
+ 
```ts
    "all:regenerate": "nx run-many -t regenerate",
```